openfasoc/generators/lc-dco/xschem_rundir/diff_cross_mirror_verilog.sch

```diff
@@ -0,0 +1,48 @@
+v {xschem version=3.0.0 file_version=1.2 }
```

Comment:
the files in xschem folder are meant as a reference? 
Normally we would build a verilog netlist, then generate its spice through openroad or yosys. 
So maybe in this case we could reuse "static" blocks directly from spice generated by xschem?